content/uc-doc/system/mobile/push_notification.md

```diff
@@ -0,0 +1,134 @@
+---
+title: 'Push Notifications'
+---
+
+The wazo-platform can generate push notifications for mobile users, through the wazo-webhookd
+component.
+
+wazo-webhookd manages subscriptions to internal platform events, such that custom behavior can be
+triggered in reaction to those events. This allows, for example, for components external to the
+wazo-platform to be notified of those events, in effect allowing for asynchronous integrations with
+other systems. See [wazo-webhookd documentation page](/uc-doc/system/wazo-webhookd.md) for more
+details.
+
+As well as supporting configuration of custom behavior through http subscriptions(created through
+the REST API) and custom python plugins, wazo-webhookd also automatically creates and manages
+special subscriptions for mobile users, in order to support push notifications for relevant
+wazo-platform events, such as calls to mobile-enabled users, voicemails, missed calls, etc.
+
+When a mobile user logs in, authentication events generated by that user login are handled by
+wazo-webhookd, which in turn creates a subscription for that user. That subscription specifies the
+internal events which are relevant for push notifications to mobile users. When those events are
+emitted by the platform, the subscription triggers the "hook" code that generates the relevant push
+notification, by interacting with the configured push notification servers.
+
+![Mobile push notifications diagram](/diagrams/mobile-mobile-push.svg)
```

Comment:
This is not intended as a sequence diagram(where arrows or direction would indicate time progression) but as a relationship diagram between the components, with the arrows representing interactions(the text beside the arrows describe the interaction). 
There is a relationship loop, since all components interact together in different roles, and indirectly through the bus.

I could remove the arrow between webhookd and wazo-auth, since that interaction is irrelevant(I copied and modified the diagram from another page).